add-on/panels/js/panel.jsx

```diff
@@ -718,22 +718,14 @@ loop.panel = (function(_, mozL10n) {
       );
     },
 
-    _renderNoRoomsView: function() {
-      return (
-        <div className="rooms">
-          {this._renderNewRoomButton()}
-          <div className="room-list-empty">
-            <div className="no-conversations-message">
-              <p className="panel-text-medium">
-                {mozL10n.get("no_conversations_message_heading2")}
-              </p>
-              <p className="panel-text-medium">
-                {mozL10n.get("no_conversations_start_message2")}
-              </p>
-            </div>
-          </div>
-        </div>
-      );
+    _renderRoomsListTitle: function() {
```

Comment:
nit: this method is alphabetically after _renderRoomsList, so put it there just before render.